### PR TITLE
Add action plugin to support dnos6_template module

### PR DESCRIPTION
**Pull request by skg-net**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME
- modules/core/network/dnos6/dnos6_template
##### ANSIBLE VERSION
- 
  ansible 2.2.0 (devel a194740) last updated 2016/09/08 17:44:28 (GMT -700) lib/ansible/modules/core: (detached HEAD db38f0c876) last updated 2016/09/08 17:44:58 (GMT -700) lib/ansible/modules/extras: (detached HEAD 8bfdcfcab2) last updated 2016/09/08 17:44:58 (GMT -700) config file = /home/skg/lothlorien/ansible.cfg configured module search path = Default w/o overrides
##### SUMMARY
- Add action plugin to support dnos6_template module

### CODE REVIEW

**Comment by skg-net** [comment]
@privateip @gundalow, please have a look at it.

File: lib/ansible/plugins/action/dnos6_template.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015 Peter Sprygada <psprygada@ansible.com>
```
Comment (gundalow) [inline lib/ansible/plugins/action/dnos6_template.py:2]: Can you please update this to you?

**Comment by gundalow** [comment]
revision_needed - Can you please apply this change to all the PRs then I'll merge. 

Thanks

**Comment by skg-net** [comment]
ready_for_review

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
needs_revision

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**